src/commonMain/kotlin/fr/acinq/lightning/io/Peer.kt

```diff
@@ -64,6 +64,11 @@ data class SendPayment(val paymentId: UUID, val amount: MilliSatoshi, val recipi
 
 data class PurgeExpiredPayments(val fromCreatedAt: Long, val toCreatedAt: Long) : PaymentCommand()
 
+internal sealed class WalletCommand : PeerCommand() {
+    data class TrySwapIn(val currentBlockHeight: Int, val wallet: WalletState, val swapInConfirmations: Int, val isMigrationFromLegacyApp: Boolean) : WalletCommand()
```

Comment:
I would have called it something more like `BalanceUpdated`.

Reply:
But it's not a balance updated event, especially if in the future we want to let the wallet retry on liquidity policy changes, it really is a request to check whether we can swap some funds in, isn't it?